packages/api/src/services/actions.ts

```diff
@@ -104,10 +104,15 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       if (addresses == null || addresses.length == 0) return queries.totalTvl(currency);
       return queries.sumTvl(addresses, currency);
     },
-    async getEmpStatsBetween(address: string, start = 0, end: number = nowS(), currency: CurrencySymbol = "usd") {
+    async getEmpStatsBetween(empAddress: string, start = 0, end: number = nowS(), currency: CurrencySymbol = "usd") {
       assert(stats[currency], "Invalid currency type: " + currency);
-      assert(stats[currency].history[address], "Invalid emp address: " + address);
-      return stats[currency].history[address].between(start, end);
+      assert(stats[currency].history[empAddress], "Invalid emp address: " + empAddress);
+      return stats[currency].history[empAddress].between(start, end);
+    },
+    async sliceHistoricalEmpStats(empAddress: string, start = 0, length = 1, currency: CurrencySymbol = "usd") {
```

Comment:
is there any bounding you think would be reasonable on this? for example a max length or a min start? is there a case where you can overload this method somehow causing slow replies? what happens if the length is more than the data available?

Reply:
yea thats a good call. there definitely needs to be some bounding, and we have to be careful about how we construct queries as well on the backend to avoid slow downs. I can add some preliminary limits, though we most likely will want to do this more thoroughly api is more stable.